zeno/util.py

```diff
@@ -241,6 +241,15 @@ def is_notebook() -> bool:
 
 
 def generate_diff_cols(df: pd.DataFrame, diff_cols: List[ZenoColumn]) -> pd.DataFrame:
```

Comment:
Since this is only ever two columns, should be two separate arguments (colA, colB) -> think about updating this for the API call as well if this is the case. Or make it a tuple with two entries

Reply:
This make sense. Will change to passing two separate arguments.